app/api/v2/handlers/schedule_api.py

```diff
@@ -0,0 +1,131 @@
+                                                                 'updated Schedule object.')
+    async def update_schedule(self, request: web.Request):
+        schedule = await self.update_object(request)
+        return web.json_response(schedule.display)
+
+    @aiohttp_apispec.docs(tags=['schedules'], summary='Replace Schedule',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'name',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'Name of the Schedule to be replaced.'
+                          }],
+                          description='Use fields from the ScheduleSchema in the request body '
+                                      'to replace an existing Schedule or create a new Schedule. '
+                                      'The name of the Schedule will be set to that of the tasked Operation.')
+    @aiohttp_apispec.request_schema(ScheduleSchema(partial=True, exclude=['name']))
+    @aiohttp_apispec.response_schema(ScheduleSchema, description='The response is a dump of the newly '
+                                                                 'replaced Schedule object.')
+    async def create_or_update_schedule(self, request: web.Request):
+        schedule = await self.create_or_update_object(request)
+        return web.json_response(schedule.display)
+
+    @aiohttp_apispec.docs(tags=['schedules'], summary='Delete Schedule',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'name',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'Name of the Schedule to be deleted.'
+                          }],
+                          description='Deletes a Schedule object from the data service.')
+    @aiohttp_apispec.response_schema(ScheduleSchema,
+                                     description='Returns HTTP 204 No Content status code if '
+                                                 'Schedule is deleted successfully.')
+    async def delete_schedule(self, request: web.Request):
+        await self.delete_object(request)
+        return web.HTTPNoContent()
+
+    '''Overridden Methods'''
+
+    async def create_object(self, request: web.Request):
+        data = await request.json()
+        data['name'] = data.get('task').get('name')
+        await self._error_if_object_with_id_exists(data.get(self.id_property))
+        access = await self.get_request_permissions(request)
+        operation = await self._api_manager.setup_operation(data['task'], access)
+        data['task'] = operation.schema.dump(operation)
+        return self._api_manager.create_object_from_schema(self.schema, data, access)
```

Comment:
Will this require us to assign the operation schema to the `task` field within the schedule object?

Reply:
We will need to dump the newly created operation object into the `task` field - from there, the `ScheduleApiManager` will create the `Schedule` object using a method from `BaseApiManager`